assets/css/structure-desktop.css

```diff
@@ -0,0 +1,45 @@
+/* Desktop */
+@media (min-width: 901px) {
+  .first-section-container {
+    display: flex;
+    justify-content: space-around;
+  }
+
+  .column-container {
+    display: flex;
+    justify-content: space-around;
+  }
+
+  .fourth-section {
+    display: flex;
+    justify-content: space-around;
+  }
+
+  .footer-container {
+    display: flex;
+    justify-content: space-around;
+  }
+}
+
+/* Tablets */
```

Comment:
There is no need to put media query for tablet breakpoint if it's the same as desktop.